dspback/triggers.py

```diff
@@ -77,6 +77,8 @@ async def watch_submissions():
                     result = await db["discovery"].delete_one({"repository_identifier": document["identifier"]})
                     logger.warning(f"delete count {result.deleted_count}")
             else:
+                # TODO: This is not going to work probably, as we don't have a document object in the case of a
```

Comment:
Please remove this TODO as #179 resolves that issue.

Reply:
Done